test/integration/user_can_edit_their_account_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.